File: SEMESTER 1/C/Cycle 3/palindrome.c
#include <stdio.h>
int main()
{
	int n, temp, r, sum=0;
	printf("Enter a number: ");
	scanf("%d", &n);
	
	temp = n;
	while(n>0)
	{
		r = n % 10;
		sum = (sum * 10) + r;
		n=n/10;
	}
	if(temp == sum)
	{
		printf("Palindrome!\n");
	}
	else
	{
		printf("Not Palindrome!\n");
	}
}

File: SEMESTER 3/PYTHON/Cycle 1/palindrome.py
n = int(input("Enter a number: "))
temp = n
sum = 0
while(n > 0):
	r = n % 10
	sum = (sum * 10) + r
	n=n // 10
    
if(temp == sum):
	print("Palindrome!\n")
else:
		print("Not Palindrome!\n")
